pkg/smart/nvme.go

```diff
@@ -0,0 +1,342 @@
+// Copyright 2017-18 Daniel Swarbrick. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
We cannot carry someone else's license in our code unless we have modified it, if we did we should mention the modifications but keep the top license as ours with citation towards the original work as a secondary license.

Reply:
Do you have an example citation?

Reply:
Did you modify this code? @wlan0 or is it just verbatim copy?

Reply:
modified